workflow/platforms/hera.yaml

```diff
@@ -128,18 +128,21 @@ platform: !Platform
 
   # Automatically detect the least used scrub area the user can access:
   least_used_temp: !Immediate 
-    - !FirstMax
-      - do: /scratch1/NCEPDEV/stmp2
-        when: !calc ( int(tools.can_write(do)) and tools.panasas_gb(do) )
-      - do: /scratch1/NCEPDEV/stmp4
-        when: !calc ( int(tools.can_write(do)) and tools.panasas_gb(do) )
-      - do: /scratch2/NCEPDEV/stmp1
-        when: !calc ( int(tools.can_write(do)) and tools.panasas_gb(do) )
-      - do: /scratch2/NCEPDEV/stmp3
-        when: !calc ( int(tools.can_write(do)) and tools.panasas_gb(do) )
-      - do: !expand "{doc.user_places.PROJECT_DIR}"
-        when: !calc tools.panasas_gb(do)/4
-        message: "{do}: use project directory as scrub space"
+    - !FirstTrue
```

Comment:
What's the difference between the FirstMax and FirstTrue?

Reply:
It's basically what it sounds like: selecting the first true value; or selecting the max value.